portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,21 +36,46 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> comments;
 
   @Override
-  public void init() {
-      comments = new ArrayList<String>();
-      comments.add("You are my loop condition. I keep coming back to you.");
-      comments.add("You are my initializer: without you, my life would point to nothing (null).");
-      comments.add("You are my semicolon; always present in everything I do.");
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String content = request.getParameter("text-input");
+    if (content.isEmpty()) {
```

Comment:
I think you may need to merge master into this branch, since I think this line was added in a previous PR.

Reply:
I've been using rebase. Is that different from merging master into a branch? Is one more preferable over the other?

Reply:
Yeah rebasing it fine also. The two have the same overall effect in the end, but there are some differences between them. The following thread describes the two if you want more info:
https://superuser.com/questions/224085/git-merge-master-into-a-branch